src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random random = new Random();
+
```

Comment:
Create here ColorSupplier field

Reply:
ColorSupplier class should be user somewhere in FigureSupplier class.